dev/io.openliberty.microprofile.health.3.1.internal_fat_tck/bnd.bnd

```diff
@@ -0,0 +1,30 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+
+src: \
+	fat/src
+
+fat.project: true
+
+tested.features=cdi-2.0,mpConfig-1.4,mpConfig-2.0
```

Comment:
no need to test against mpConfig-1.4, not a supported combination

Reply:
Removed mpConfig-1.4 feature from begin tested.